lib/folio_client/job_status.rb

```diff
@@ -19,13 +19,11 @@ def initialize(client, job_execution_id:)
 
     # @return [Dry::Monads::Result] Success if job is complete,
     # Failure(:pending) if job is still running,
-    # Failure(:error) if job has errors
     # Failure(:not_found) if job is not found
     def status
-      response_hash = client.get("/metadata-provider/jobSummary/#{job_execution_id}")
+      response_hash = client.get("/change-manager/jobExecutions/#{job_execution_id}")
 
-      return Failure(:error) if response_hash["totalErrors"].positive?
-      return Failure(:pending) if response_hash.dig("sourceRecordSummary", "totalCreatedEntities").zero? && response_hash.dig("sourceRecordSummary", "totalUpdatedEntities").zero?
+      return Failure(:pending) if !["COMMITTED", "ERROR"].include?(response_hash["status"])
```

Comment:
An "ERROR" approach means one or more records failed, but it does not mean they all fail. We probably need a more nuanced way to handle this eventually. Given there are zero consumers of this part of the Ruby gem, though, I was thinking it'd be fine to say "this is good enough for now."

Reply:
Put in a comment to this effect?

Reply:
Good call, done.